src/euler3d_mappedGrid.f90

```diff
@@ -0,0 +1,679 @@
+MODULE mg
+
+  REAL(kind=8), PARAMETER :: pi=3.1415926535897932385,twopi=2*pi
+  REAL(kind=8), PARAMETER :: rEarth=637120000.0
```

Comment:
Ahh, yes sorry for the confusion there.  I've been working in cgs, so the
Earth radius is in centimeters.

You're probably right, better to just explicitly declare as double
precision, that way with different compilers it does not get screwed up.

On Tue, May 5, 2015 at 10:51 AM, Kyle Mandli <notifications@github.com>
wrote:

> In src/euler3d_mappedGrid.f90
> <https://github.com/clawpack/riemann/pull/85#discussion_r29675603>:
>
> > @@ -0,0 +1,679 @@
> > +MODULE mg
> > +
> > +  REAL(kind=8), PARAMETER :: pi=3.1415926535897932385,twopi=2*pi
> > +  REAL(kind=8), PARAMETER :: rEarth=637120000.0
>
> I am not sure if that's a good assumption to make, at the very least what
> you will get I think is the single-precision representation put into a
> double precision variable.
>
> Beyond that I am puzzled by the magnitude of the radius of the Earth here,
> not necessarily the exact number. It seems like you are off by a factor of
> 1000?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clawpack/riemann/pull/85/files#r29675603>.
>